<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoBox</class>
 <widget class="QWidget" name="VideoBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>257</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>257</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VideoBox{
    background-color : #FFFFFF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="imageBox" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>135</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="videoInfoBox" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>46</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>46</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    color : #FFFFFF;
    font-size : 14px;
    font-family : 微软雅黑
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="playBox" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="playLabel">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#playLabel{
    border-image : url(&quot;:/images/homePage/bofang.png&quot;);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="playNum">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="likeBox" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="likeImage">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#likeImage{
    border-image : url(&quot;:/images/homePage/dainzan.png&quot;);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="likeNum">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="timeBox" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="videoDuration">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="videoTittleBox" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="videoTittle">
        <property name="styleSheet">
         <string notr="true">#videoTittle{
    color : #666666;
    font-weight : bold;
    font-size : 16px;
    font-family : 微软雅黑;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="userBox" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>19</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
     color : #999999;
     font-size : 14px;
     font-family : 微软雅黑;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="userIcon">
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userNickName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="loadupTime">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>160</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="delVideoBtn">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#delVideoBtn{
    border : none;
    background : transparent;
    border-image : url(&quot;:/images/mySelf/gengduo.png&quot;);
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
